sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat.py

```diff
@@ -53,19 +84,33 @@ def collect_statsbeat_metrics(exporter) -> None:
                 long_interval_threshold,
                 exporter._credential is not None,
                 exporter._distro_version,
-            )
-        # Export some initial stats on program start
+            )        # Export some initial stats on program start
         mp.force_flush()
```

Comment:
If we are using `_delayed_export_statsbeat` as the initial export, we need to remove this line.

Reply:
Force flush removed.